app/helpers/languages_helper.rb

```diff
@@ -199,6 +199,8 @@ module LanguagesHelper
     sco: ['Scots', 'Scots'].freeze,
     sma: ['Southern Sami', 'Åarjelsaemien Gïele'].freeze,
     smj: ['Lule Sami', 'Julevsámegiella'].freeze,
+    szl: ['Silesian', 'ślůnsko godka'].freeze,
```

Comment:
These and the next value I got from Wiki/ISO searching and may not be truly correct